splitio/push/status_tracker.py

```diff
@@ -145,16 +151,20 @@ def _update_status(self):
         if self._last_status_propagated == Status.PUSH_SUBSYSTEM_UP:
             if not self._occupancy_ok() \
                     or self._last_control_message == ControlType.STREAMING_PAUSED:
+                self._telemetry_runtime_producer.record_streaming_event(('STREAMING_STATUS', 'PAUSED', self._timestamps))
```

Comment:
Not so important, but consider defining string literals as constants to reuse across the code.

Reply:
will do